### PR TITLE
Update learn-treasury.md

### PR DESCRIPTION
**Pull request by rrtti**
Add bounty spending mechanism information as the third treasury spending mechanism.

### CODE REVIEW

**Comment by Swader** [comment]
Remember to run prettier to make CI pass. `npx run prettier --save` should do I think.

**Comment by lsaether** [comment]
@rrtti Is this ready for review?

**Comment by Swader** [comment]
I think we're waiting for implementation to finish so we can test drive on a dev chain and get screenshots for the guide

**Review by rrtti — COMMENTED** [review]
Ready for review.

File: docs/learn-treasury.md [hunk 1]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
 In order for Bob to be paid his tip, some account must call the `close_tip` extrinsic at the end of
 the closing period for the tip. This extrinsic may be called by anyone.
 
+## Bounties Spending
+
+There are practical limits to Council Members curation capabilities when it comes to treasury proposals: Council members likely do not have the expertise to make a proper assessment of the activities described in all proposals. Even if individual Councillors have that expertise, it is highly unlikely that a majority of members are capable in such diverse topics.
+
+Bounties Spending proposals aim to delegate the curation activity of spending proposals to experts called Curators: They can be defined as addresses with agency over a portion of the treasury with the goal of fixing a bug, vulnerability, develop a strategy or monitoring a set of tasks related to a specific topic: all for the benefit of the Polkadot ecosystem.
```
Comment (Swader) [inline docs/learn-treasury.md]:
```suggestion
Bounties Spending proposals aim to delegate the curation activity of spending proposals to experts called Curators: They can be defined as addresses with agency over a portion of the treasury with the goal of fixing a bug, vulnerability, develop a strategy, or monitoring a set of tasks related to a specific topic: all for the benefit of the Polkadot ecosystem. 
```

File: docs/learn-treasury.md [hunk 2]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+Bounties Spending proposals aim to delegate the curation activity of spending proposals to experts called Curators: They can be defined as addresses with agency over a portion of the treasury with the goal of fixing a bug, vulnerability, develop a strategy or monitoring a set of tasks related to a specific topic: all for the benefit of the Polkadot ecosystem. 
+
+A proposer can submit a bounty proposal for the Council to pass, with a curator to be defined after whose background and expertise is such that they are capable of determining when the task is complete. Curators are selected by the Council after the bounty proposal passes, and need to add an upfront payment to take the position. This deposit can be used to punish them if they act maliciously. But if they are successful in their task of getting someone to complete the bounty work, they will receive their deposit back and part of the bounty reward.
```
Comment (Swader) [inline docs/learn-treasury.md]:
```suggestion
A proposer can submit a bounty proposal for the Council to pass, with a curator to be defined later, whose background and expertise is such that they are capable of determining when the task is complete. Curators are selected by the Council after the bounty proposal passes, and need to add an upfront payment to take the position. This deposit can be used to punish them if they act maliciously. But if they are successful in their task of getting someone to complete the bounty work, they will receive their deposit back and part of the bounty reward.
```

File: docs/learn-treasury.md [hunk 3]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+The bounty has a predetermined duration of 90 days with the possibility of being extended by the curator. Aiming to maintain flexibility on the tasks’ curation, the curator will be able to create sub-bounties for more granularity and allocation in the next iteration of the mechanism.    
+
+### Creating a Bounty Proposal
+
+Anyone can create a Bounty proposal using Polkadot JS Apps: From the website, use the Extrinsics tab and select the Treasury pallet, then `Propose_bounty` and enter the value and description of the proposal (usually linked to a forum). In the near future, users can use the dedicated Bounty section under Governance. The development of a robust user interface to view and manage bounties in the Polkadot Apps is on its way: It will serve Council members, Curators and Beneficiaries of the bounties, as well as all Kusama users observing the on-chain treasury governance. For now,
```
Comment (Swader) [inline docs/learn-treasury.md]: Unfinished sentence at the end?

File: docs/learn-treasury.md [hunk 4]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+The bounty has a predetermined duration of 90 days with the possibility of being extended by the curator. Aiming to maintain flexibility on the tasks’ curation, the curator will be able to create sub-bounties for more granularity and allocation in the next iteration of the mechanism.    
+
+### Creating a Bounty Proposal
+
+Anyone can create a Bounty proposal using Polkadot JS Apps: From the website, use the Extrinsics tab and select the Treasury pallet, then `Propose_bounty` and enter the value and description of the proposal (usually linked to a forum). In the near future, users can use the dedicated Bounty section under Governance. The development of a robust user interface to view and manage bounties in the Polkadot Apps is on its way: It will serve Council members, Curators and Beneficiaries of the bounties, as well as all Kusama users observing the on-chain treasury governance. For now, 
+
+A bounty can be cancelled by deleting the earmark for a specific treasury amount of be closed if the tasks have been completed. On the opposite side, the 90 days life of a bounty can be extended by amending the expiry block number of the bounty to stay active.
```
Comment (Swader) [inline docs/learn-treasury.md]:
```suggestion
A bounty can be cancelled by deleting the earmark for a specific treasury amount or be closed if the tasks have been completed. On the opposite side, the 90 days life of a bounty can be extended by amending the expiry block number of the bounty to stay active. 
```

File: docs/learn-treasury.md [hunk 5]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+### Creating a Bounty Proposal
+
+Anyone can create a Bounty proposal using Polkadot JS Apps: From the website, use the Extrinsics tab and select the Treasury pallet, then `Propose_bounty` and enter the value and description of the proposal (usually linked to a forum). In the near future, users can use the dedicated Bounty section under Governance. The development of a robust user interface to view and manage bounties in the Polkadot Apps is on its way: It will serve Council members, Curators and Beneficiaries of the bounties, as well as all Kusama users observing the on-chain treasury governance. For now, 
+
+A bounty can be cancelled by deleting the earmark for a specific treasury amount of be closed if the tasks have been completed. On the opposite side, the 90 days life of a bounty can be extended by amending the expiry block number of the bounty to stay active. 
+
+### Closing a bounty
+
+The curator can close the bounty once it approves the completion of its tasks. The curator should make sure to set up the payout address on the active bounty beforehand. Closing the Active bounty enacts a delayed payout to the payout address and a payout of the curator fee.
```
Comment (Swader) [inline docs/learn-treasury.md]:
```suggestion
The curator can close the bounty once they approve the completion of its tasks. The curator should make sure to set up the payout address on the active bounty beforehand. Closing the Active bounty enacts a delayed payout to the payout address and a payout of the curator fee.
```

File: docs/learn-treasury.md [hunk 6]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+A bounty can be closed by using the extrinsics tab and selecting the Treasury pallet, then `Award_bounty`, making sure the right bounty is to be closed and finally sign the transaction. It is important to note that those who received a reward after the bounty is completed, must claim the specific amount of the payout from the payour address, by calling `Claim_bounty` after the curator closed the allocation.
```
Comment (Swader) [inline docs/learn-treasury.md]:
```suggestion
A bounty can be closed by using the extrinsics tab and selecting the Treasury pallet, then `Award_bounty`, making sure the right bounty is to be closed and finally sign the transaction. It is important to note that those who received a reward after the bounty is completed, must claim the specific amount of the payout from the payout address, by calling `Claim_bounty` after the curator closed the allocation. 
```

File: docs/learn-treasury.md [hunk 7]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+### Closing a bounty
+
+The curator can close the bounty once it approves the completion of its tasks. The curator should make sure to set up the payout address on the active bounty beforehand. Closing the Active bounty enacts a delayed payout to the payout address and a payout of the curator fee.
+
+A bounty can be closed by using the extrinsics tab and selecting the Treasury pallet, then `Award_bounty`, making sure the right bounty is to be closed and finally sign the transaction. It is important to note that those who received a reward after the bounty is completed, must claim the specific amount of the payout from the payour address, by calling `Claim_bounty` after the curator closed the allocation. 
+
+To understand more about Bounties and how this new mechanism works, visit [Polkadot Blog](https://polkadot.network/kusama-and-polkadot-now-reward-curators-helping-to-scale-councils-functions-join-the-force-moving-the-community-forward/).
```
Comment (Swader) [inline docs/learn-treasury.md]:
```suggestion
To understand more about Bounties and how this new mechanism works, read this [Polkadot Blog post](https://polkadot.network/kusama-and-polkadot-now-reward-curators-helping-to-scale-councils-functions-join-the-force-moving-the-community-forward/). 
```

**Review by Swader — CHANGES_REQUESTED** [review]
Nitpicks

**Comment by laboon** [comment]
What is the status of this?  @rrtti

**Comment by rrtti** [comment]
@laboon I have edited the content in the last committed version for users to understand they need help from a council member until the UI is ready. Once the UI Bounty is closed we will have screenshots of where the user goes to submit one on Polkadot.js Apps, for now i have edited the content for a temporary use (see "### Creating a Bounty Proposal"). 

Let me know if that works, we wont have a screenshot for a few weeks at least.

**Comment by laboon** [comment]
@rrtti So is it ready for re-review?  It's still listed as draft.

File: docs/learn-treasury.md [hunk 8]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
 In order for Bob to be paid his tip, some account must call the `close_tip` extrinsic at the end of
 the closing period for the tip. This extrinsic may be called by anyone.
 
+## Bounties Spending
+
+There are practical limits to Council Members curation capabilities when it comes to treasury proposals: Council members likely do not have the expertise to make a proper assessment of the activities described in all proposals. Even if individual Councillors have that expertise, it is highly unlikely that a majority of members are capable in such diverse topics.
```
Comment (laboon) [inline docs/learn-treasury.md:166]:
```suggestion
There are practical limits to Council Members curation capabilities when it comes to Treasury proposals: Council members likely do not have the expertise to make a proper assessment of the activities described in all proposals. Even if individual Councillors have that expertise, it is highly unlikely that a majority of members are capable in such diverse topics.
```

File: docs/learn-treasury.md [hunk 9]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+Bounties Spending proposals aim to delegate the curation activity of spending proposals to experts called Curators: They can be defined as addresses with agency over a portion of the treasury with the goal of fixing a bug, vulnerability, develop a strategy, or monitoring a set of tasks related to a specific topic: all for the benefit of the Polkadot ecosystem.
```
Comment (laboon) [inline docs/learn-treasury.md:168]:
```suggestion
Bounties spending proposals aim to delegate the curation activity of spending proposals to experts called Curators. Curators are defined as addresses with agency over a portion of the treasury with the goal of fixing a bug, vulnerability, develop a strategy, or monitoring a set of tasks related to a specific topic for the benefit of the Polkadot ecosystem. 
```

File: docs/learn-treasury.md [hunk 10]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+Bounties Spending proposals aim to delegate the curation activity of spending proposals to experts called Curators: They can be defined as addresses with agency over a portion of the treasury with the goal of fixing a bug, vulnerability, develop a strategy, or monitoring a set of tasks related to a specific topic: all for the benefit of the Polkadot ecosystem. 
+
+A proposer can submit a bounty proposal for the Council to pass, with a curator to be defined later, whose background and expertise is such that they are capable of determining when the task is complete. Curators are selected by the Council after the bounty proposal passes, and need to add an upfront payment to take the position. This deposit can be used to punish them if they act maliciously. But if they are successful in their task of getting someone to complete the bounty work, they will receive their deposit back and part of the bounty reward.
```
Comment (laboon) [inline docs/learn-treasury.md:170]:
```suggestion
A proposer can submit a bounty proposal for the Council to pass, with a curator to be defined later, whose background and expertise is such that they are capable of determining when the task is complete. Curators are selected by the Council after the bounty proposal passes, and need to add an upfront deposit to take the position. This deposit can be revoked if the curator is found to have acted maliciously.  If they are successful in their task of getting someone to complete the bounty work, they will receive their deposit back as well as the bounty reward.
```

File: docs/learn-treasury.md [hunk 11]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+When submitting the value of the bounty, the proposer includes a reward for curators willing to invest their time and expertise in the task: this amount is included in the total value of the bounty. In this sense, the curator's fee can be defined as the result of subtracting the value paid to the bounty rewardee from the total value of the bounty.
```
Comment (laboon) [inline docs/learn-treasury.md:172]:
```suggestion
When submitting the value of the bounty, the proposer includes a reward for curators willing to invest their time and expertise in the task. This amount is included in the total value of the bounty. The curator's fee can thus be defined as the result of subtracting the value paid to the bounty rewardee from the total value of the bounty.
```

File: docs/learn-treasury.md [hunk 12]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+In general terms, curators are expected to have a well-balanced track record related to the issues the bounty tries to resolve: they should be at least knowledgeable on the topics the bounty touches, and show project management skills or experience. These recommendations ensure an effective use of the mechanism. A Bounty Spending is a reward for a specified body of work - or specified set of objectives - that needs to be executed for a predefined treasury amount to be paid out. The responsibility of assigning a payout address once the specified set of objectives is completed is delegated to the curator.
```
Comment (laboon) [inline docs/learn-treasury.md:174]:
```suggestion
In general terms, curators are expected to have a well-balanced track record related to the issue the bounty tries to resolve. They should be knowledgeable on the topics the bounty touches, and ideally also show project management skills or experience. A Bounty Spending is a reward for a specified body of work - or specified set of objectives - that needs to be executed for a predefined Treasury amount to be paid out. The responsibility of assigning a payout address once the specified set of objectives is completed is delegated to the curator.
```

File: docs/learn-treasury.md [hunk 13]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+After the Council has activated a bounty, it delegates the work that requires expertise to the curator who gets to close the active bounty. Closing the active bounty enacts a delayed payout to the payout address and a payout of the curator fee. The delay phase allows the Council to act if any issues arise.
```
Comment (laboon) [inline docs/learn-treasury.md:176]:
```suggestion
After the Council has activated a bounty, it delegates the work that requires expertise to the curator who gets to close the active bounty. Closing the active bounty enacts a delayed payout to the payout address and a payout of the curator fee. The delay phase allows the Council to act if any issues arise immediately after payment.
```

File: docs/learn-treasury.md [hunk 14]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+To minimize storage on chain in the same way as any proposal, bounties don't contain contextual information. When a user submits a bounty spending proposal, they will probably need to find an off-chain way to explain the proposal (any of the available community forums serve this purpose). [This template](https://docs.google.com/document/d/1-IBz_owspV5OcvezWXpksWDQReWowschD0TFuaVKKcU/edit?usp=sharing) can help as a checklist of all needed information for the Council to make an informed decision.
```
Comment (laboon) [inline docs/learn-treasury.md:178]:
```suggestion
To minimize storage on chain in the same way as any proposal, bounties don't contain contextual information. When a user submits a bounty spending proposal, they will probably need to find an off-chain way to explain the proposal (any of the [available community forums](community) serve this purpose). [This template](https://docs.google.com/document/d/1-IBz_owspV5OcvezWXpksWDQReWowschD0TFuaVKKcU/edit?usp=sharing) can help as a checklist of all needed information for the Council to make an informed decision.
```

Comment (laboon) [reply docs/learn-treasury.md:178]: Ideally we'd move the template to IPFS or some other more permanent way to access it but it's not necessary.

File: docs/learn-treasury.md [hunk 15]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+The bounty has a predetermined duration of 90 days with the possibility of being extended by the curator. Aiming to maintain flexibility on the tasks’ curation, the curator will be able to create sub-bounties for more granularity and allocation in the next iteration of the mechanism.
```
Comment (laboon) [inline docs/learn-treasury.md:180]:
```suggestion
The bounty has a default duration of 90 days with the possibility of being extended by the curator. In the future, the curator will be able to create sub-bounties for more granularity and fine-grained allocation.
```

File: docs/learn-treasury.md [hunk 16]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+The bounty has a predetermined duration of 90 days with the possibility of being extended by the curator. Aiming to maintain flexibility on the tasks’ curation, the curator will be able to create sub-bounties for more granularity and allocation in the next iteration of the mechanism.    
+
+### Creating a Bounty Proposal
+
+Anyone can create a Bounty proposal using Polkadot JS Apps: Users will be able to submit a proposal on the dedicated Bounty section under Governance. The development of a robust user interface to view and manage bounties in the Polkadot Apps is on its way and it will serve Council members, Curators and Beneficiaries of the bounties, as well as all Kusama users observing the on-chain treasury governance. For now, the help of a Councillor is needed to open a bounty proposal as a motion to be voted.
```
Comment (laboon) [inline docs/learn-treasury.md:184]:
```suggestion
Soon, anyone will be able to create a Bounty proposal using Polkadot-JS. Users will be able to submit a proposal on the dedicated [Bounty](https://polkadot.js.org/apps/#/bounties) page. Currently, this page can only be used to view active bounties. The development of a more interactive user interface to both view and manage bounties in Polkadot-JS is currently being worked on, itself as a bounty. For now, the help of a Councillor is needed to open a bounty proposal as a motion on which to be voted. 
```

File: docs/learn-treasury.md [hunk 17]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+The bounty has a predetermined duration of 90 days with the possibility of being extended by the curator. Aiming to maintain flexibility on the tasks’ curation, the curator will be able to create sub-bounties for more granularity and allocation in the next iteration of the mechanism.    
+
+### Creating a Bounty Proposal
+
+Anyone can create a Bounty proposal using Polkadot JS Apps: Users will be able to submit a proposal on the dedicated Bounty section under Governance. The development of a robust user interface to view and manage bounties in the Polkadot Apps is on its way and it will serve Council members, Curators and Beneficiaries of the bounties, as well as all Kusama users observing the on-chain treasury governance. For now, the help of a Councillor is needed to open a bounty proposal as a motion to be voted. 
+
+A bounty can be cancelled by deleting the earmark for a specific treasury amount or be closed if the tasks have been completed. On the opposite side, the 90 days life of a bounty can be extended by amending the expiry block number of the bounty to stay active.
```
Comment (laboon) [inline docs/learn-treasury.md:186]:
```suggestion
A bounty can be cancelled by deleting the earmark for a specific treasury amount, or closed if the task has been completed. The period for a bounty to be completed can also be extended by amending the expiry block number of the bounty to a higher block number. 
```

File: docs/learn-treasury.md [hunk 18]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+### Creating a Bounty Proposal
+
+Anyone can create a Bounty proposal using Polkadot JS Apps: Users will be able to submit a proposal on the dedicated Bounty section under Governance. The development of a robust user interface to view and manage bounties in the Polkadot Apps is on its way and it will serve Council members, Curators and Beneficiaries of the bounties, as well as all Kusama users observing the on-chain treasury governance. For now, the help of a Councillor is needed to open a bounty proposal as a motion to be voted. 
+
+A bounty can be cancelled by deleting the earmark for a specific treasury amount or be closed if the tasks have been completed. On the opposite side, the 90 days life of a bounty can be extended by amending the expiry block number of the bounty to stay active. 
+
+### Closing a bounty
+
+The curator can close the bounty once they approve the completion of its tasks. The curator should make sure to set up the payout address on the active bounty beforehand. Closing the Active bounty enacts a delayed payout to the payout address and a payout of the curator fee.
```
Comment (laboon) [inline docs/learn-treasury.md:190]:
```suggestion
The curator can close the bounty once they approve the completion of all of the associated tasks. The curator should be sure to set up the payout address on the active bounty beforehand. Closing an active bounty enacts a delayed payout to the payout address as well as the payout of the curator fee.
```

File: docs/learn-treasury.md [hunk 19]
```diff
@@ -161,6 +161,38 @@ Now the actual tip that will be paid out to Bob is the median of these tips, so
...
+
+A bounty can be closed by using the extrinsics tab and selecting the Treasury pallet, then `Award_bounty`, making sure the right bounty is to be closed and finally sign the transaction. It is important to note that those who received a reward after the bounty is completed, must claim the specific amount of the payout from the payout address, by calling `Claim_bounty` after the curator closed the allocation.
```
Comment (laboon) [inline docs/learn-treasury.md:192]:
```suggestion
A bounty can be closed by using the [extrinsics](https://polkadot.js.org/apps/#/extrinsics) page and selecting `treasury.awardBounty`, setting the correct bounty ID and beneficiary, and then submitting and signing the extrinsic. The account that received a reward after the bounty is completed must claim the payout themself, by calling `treasury.claimBounty` after the curator has awarded the bounty. 
```

**Review by laboon — CHANGES_REQUESTED** [review]

**Comment by laboon** [comment]
Raul - I made some wording changes, please do this and double-check formatting and links.

I vote we get this in now and I can add screenshots later.

**Comment by rrtti** [comment]
@laboon: Sounds good, all resolved - if you want to add a screenshot already for it to be there the first iteration of a UI is on Apps ready for users to see.

**Review by rrtti — COMMENTED** [review]
All good from my PoV.

File: docs/learn-treasury.md (hunk 14 above)
Comment (rrtti) [reply docs/learn-treasury.md:178]: do we need to add a link here for the community forums?

**Comment by laboon** [comment]
Merged in these changes as PR #1431

**Pull request closed without merging**